app/assets/javascripts/discourse/app/lib/sidebar/helpers.js

```diff
@@ -15,3 +15,11 @@ export function hasDefaultSidebarCategories(siteSettings) {
 export function hasDefaultSidebarTags(siteSettings) {
   return siteSettings.default_navigation_menu_tags.length > 0;
 }
+
+export function getSidebarSectionContentID(name) {
```

Comment:
minor naming note - we usually use `Id` vs `ID` in names (did a regex search in core to check my gut feeling and it's 3077 vs 97 hits 😃)

Reply:
I changed to `getSidebarSectionContentId`